packages/components/src/Enumeration/Header/HeaderInput.component.js

```diff
@@ -24,7 +24,9 @@ function getAction(action, index, getInternalInputRef) {
 			});
 		}
 	}
-
+	if (action.disabled) {
+		return null;
+	}
```

Comment:
Same remark  : You should filter the actions in the main component `return`, before the map()